omaha/base/constants.h

```diff
@@ -174,7 +176,7 @@ const TCHAR* const kChromeAppId = CHROME_APP_ID;
 #define INSTALL_WORKING_DIR_NAME  _T("Install")
 
 // Directories relative to \Google
-#define OMAHA_REL_COMPANY_DIR SHORT_COMPANY_NAME
+#define OMAHA_REL_COMPANY_DIR REG_KEY_NAME
```

Comment:
For us, the constants are defined as follows:

```
_FULL_COMPANY_NAME = 'Kings Distributed Systems'
_SHORT_COMPANY_NAME = 'KDS'
_PATH_COMPANY_NAME = 'Kings Distributed Systems'
```

The comments describing the constants, and how they are differentiated:

```
// Full company name.
// FULL_COMPANY_NAME == "Google LLC"
#define FULL_COMPANY_NAME _T(FULL_COMPANY_NAME_ANSI)
const TCHAR* const kFullCompanyName = FULL_COMPANY_NAME;

// Short company name (for use in messages and to combine with product name).
// Does not include "Inc." and similar formal parts of the company name.
// If the company name can be abbreviated, it will be here.
// SHORT_COMPANY_NAME == "Google"
#define SHORT_COMPANY_NAME _T(SHORT_COMPANY_NAME_ANSI)
const TCHAR* const kShortCompanyName = SHORT_COMPANY_NAME;

// The company name to use in file and registry paths.
// PATH_COMPANY_NAME == "Google"
#define PATH_COMPANY_NAME _T(PATH_COMPANY_NAME_ANSI)
const TCHAR* const kPathCompanyName = PATH_COMPANY_NAME;
```